composer.js

```diff
@@ -199,8 +199,8 @@ function composeJoint(params){
 	//profiler.start();
 	var arrChangeOutputs = arrOutputs.filter(function(output) { return (output.amount === 0); });
 	var arrExternalOutputs = arrOutputs.filter(function(output) { return (output.amount > 0); });
-	if (arrChangeOutputs.length > 1)
-		throw Error("more than one change output");
+	// if (arrChangeOutputs.length > 1)
```

Comment:
More than 1 change output doesn't make sense, you should not do your task at the expense of breaking the existing sanity checks.